ADOPTERS.md

```diff
@@ -4,6 +4,12 @@ If you're using Tanzu Community Edition and want to add your organization to thi
 
 ## Organizations using Tanzu Community Edition
 
+<a href="https://beam.lu/" border="0" target="_blank"><img alt="Beam" src="logos/terasky.png" height="50"></a>
```

Comment:
You can change this to:

```md
![terasky logo](logos/terasky.png)
```

Or to have it be a link (probably not to beam.lu though), I think you can do:

```md
[![terasky logo](logos/terasky.png)](https://terasky.com/)

Reply:
awesome just pushed the change. thanks for the help!